.github/workflows/ci_ol8.yml

```diff
@@ -0,0 +1,58 @@
+name: CI-cpp-ol8
+
+on:
+  workflow_dispatch: {}
+  pull_request:
+    types: [opened, ready_for_review, reopened]
+  push:
+    paths:
+      - 'metrix-simulator/**'
+      - '.github/workflows/ci_ol8.yml'
+  release:
+    types: [published]
+
+# Cancel previous workflows if they are the same workflow on same ref (branch/tags)
+# with the same event (push/pull_request) even they are in progress.
+# This setting will help reduce the number of duplicated workflows.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  linux:
+    name: Tests C++ Oracle Linux 8
+    runs-on: ubuntu-latest
+    container: 'oraclelinux:8'
+    steps:
+      - name: Install Boost
+        run: |
+          yum update -y
+          yum install cmake make gcc gcc-c++ which git
+          dnf --enablerepo=ol8_codeready_builder install boost-static
+
+      - name: Checkout sources
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
Is it on purpose that you don't always use the last version for the actions (version `4.1.4` exists for `actions/checkout`, but the version for `actions/upload-artifact` is up-to-date)?

Reply:
Nope, I used the same version as Core, but I'll update this